src/os/os.c

```diff
@@ -918,7 +918,7 @@ int ignore_fn(void)
     return 0;
 }
 
-#if !defined(ESP32)
+#if !defined(ARDUINO)
```

Comment:
Can we go with:
```#if !defined(ARUDINO) && !defined(ESP32)```
Since I no longer have arduino defined in my CS build this may break when I next sync.

Reply:
done

Reply:
Thanks